<launch>
  <arg name="debug" default="false" />
  <arg name="use_gui" default="false" />

  <!-- Load the URDF, SRDF and other .yaml configuration files on the param server -->
  <include file="$(find victor_moveit_config)/launch/planning_context.launch">
    <arg name="load_robot_description" value="true"/>
  </include>

  <!--&lt;!&ndash; Run the main MoveIt executable without trajectory execution (we do not have controllers configured by default) &ndash;&gt;-->
  <!--<include file="$(find victor_moveit_config)/launch/move_group.launch">-->
    <!--<arg name="allow_trajectory_execution" value="true"/>-->
    <!--<arg name="fake_execution" value="true"/>-->
    <!--<arg name="info" value="true"/>-->
    <!--<arg name="debug" value="$(arg debug)"/>-->
  <!--</include>-->

  <node pkg="dual_arm_teleop"
        type="dual_arm_teleop_node"
        name="dual_arm_teleop_node"
        respawn="false" output="screen">
    <rosparam command="load"
              file="$(find victor_moveit_config)/config/kinematics.yaml"/>
  </node>
</launch>